src/server/models/cacheable_queries/question-response.js

```diff
@@ -50,8 +50,12 @@ const questionResponseCache = {
   save: async (campaignContactId, questionResponses) => {
     // This is a bit elaborate because we want to preserve the created_at time
     // Otherwise, we could just delete all and recreate
+    const toReturn = {
+      newOrUpdatedPreviousValue: {},
+      deletedPrevious: []
```

Comment:
nit: these keys would be simpler as just `newOrUpdated` and `deleted` -- I don't know what Previous and then Value is at the end of one but not the other?